test/behavior/zon/slice-1.zon

```diff
@@ -0,0 +1 @@
+.{ 1 }
```

Comment:
missing trailing newlines?

Reply:
That's intentional, I wanted at least one test case to cover missing a trailing new line. Even if it's not recommended it's technically allowed afaik, I'll rename the file to make clear that it's intentional